smartsim/settings/pbsSettings.py

```diff
@@ -66,9 +68,18 @@ def __init__(
         :param batch_args: overrides for PBS batch arguments, defaults to None
         :type batch_args: dict[str, str], optional
         """
-        self._time: t.Optional[str] = None
-        self._nodes: t.Optional[int] = None
+
         self._ncpus = ncpus
+        self.resources: dict[str, t.Union[str,int]] = {}
```

Comment:
Unfortunently this:
```py
self.resources: dict[str, t.Union[str,int]] = {}
#               ^^^^
```
is only parsable w/o setting the `annotations` flag with python>=3.9. We need to make this `t.Dict` :(

Reply:
beat me to it, hahaha